OpenDreamShared/Dream/Procs/ProcAttributes.cs

```diff
@@ -4,7 +4,6 @@
 namespace OpenDreamShared.Dream.Procs;
 
 [Flags]
-[DefaultValue(None)]
```

Comment:
This doesn't actually seem to be necessary.

Reply:
The default value of an enum is whichever has a value of `0`. There is no value here that is zero because `None` is `1 << 0` which is `1`.

Reply:
This leads the default value of this enum to be `(ProcAttributes)0` because there is no value of `ProcAttributes` equal to zero. `ProcAttributes.None` is `1 << 0` which is `1`.

Reply:
Is it fine to just set None to 0? Because I did.

Reply:
Yep